src/Resources/app/administration/src/module/sw-in-app-purchases/component/sw-in-app-purchase-checkout-overview/sw-in-app-purchase-checkout-overview.scss

```
@@ -31,6 +31,25 @@
             font-size: var(--font-size-s);
             color: var(--color-text-secondary-default);
         }
+
+        &__choices {
+            margin-bottom:0;
+
+            .sw-field__radio-group{
+                .sw-field__radio-option {
+                    padding:var(--scale-size-24);
+
+                    .sw-field__radio-input {
+                        margin-top:var(--scale-size-x);
```

I didn't mean the literal `--scale-size-x` but a scale appropriate to the margin you wanted to have :joy:
```suggestion
                        margin-top:var(--scale-size-24);
```

---

holy crap... WTF was I doing 😂